src/views/hardware_view.rs

```diff
@@ -486,14 +486,14 @@ fn get_pin_widget(
             .size(TOGGLER_SIZE)
             .style(toggle_button_style.get_toggler_style());
 
-            let output_clicker = mouse_area(clicker(BUTTON_WIDTH))
+            let output_clicker = clicker::<HardwareMessage, Renderer>(BUTTON_WIDTH)
                 .on_press({
                     let level: PinLevel = pin_state.get_level().unwrap_or(false as PinLevel);
-                    ChangeOutputLevel(bcm_pin_number.unwrap(), LevelChange::new(!level))
+                    HardwareMessage::ChangeOutputLevel(bcm_pin_number.unwrap(), LevelChange::new(!level))
```

Comment:
Did you add the prefix deliberately to make clearer?
My IDE tends to suggest the import of the suffix directly, so the prefix (HardwareMessage) is not needed and the code is a bit shorter.